# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/nr_socket_tcp.cpp
# Commit: da41f0fab427
# Full Hash: da41f0fab427b199f1a0c6156467a0ad9a9e6891
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Make proxy config parameters optional in WebrtcTCPSocket to get the NAT simulator working again. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45104
# ==============================================================================

diff -r b83124cf3a89 -r da41f0fab427 media/mtransport/nr_socket_tcp.cpp
--- a/media/mtransport/nr_socket_tcp.cpp	Tue Sep 17 18:17:24 2019 +0000
+++ b/media/mtransport/nr_socket_tcp.cpp	Tue Sep 17 18:17:37 2019 +0000
@@ -70,7 +70,6 @@
       mConfig(aConfig),
       mWebrtcTCPSocket(nullptr) {
   r_log(LOG_GENERIC, LOG_DEBUG, "NrTcpSocket::NrTcpSocket %p\n", this);
-  MOZ_ASSERT(mConfig, "config should not be null");
 }
 
 NrTcpSocket::~NrTcpSocket() {